react/features/large-video/subscriber.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+import { StateListenerRegistry } from '../base/redux';
+import VideoLayout from '../../../modules/UI/videolayout/VideoLayout';
+
+/**
+ * Updates the on stage participant video.
+ */
+StateListenerRegistry.register(
+    /* selector */ state => state['features/large-video'].participantId,
+    /* listener */ participantId => {
+        VideoLayout.updateLargeVideo(participantId, true);
```

Comment:
This change just broke mobile. `VideoLayout` is not working there.

Reply:
Ouchh... Sorry! Will fix it!